includes/widgets/traits/button.php

```diff
@@ -0,0 +1,504 @@
+<?php
+
+use Elementor\Controls_Manager;
+use Elementor\Core\Kits\Documents\Tabs\Global_Colors;
+use Elementor\Core\Kits\Documents\Tabs\Global_Typography;
+use Elementor\Group_Control_Background;
+use Elementor\Group_Control_Border;
+use Elementor\Group_Control_Box_Shadow;
+use Elementor\Group_Control_Text_Shadow;
+use Elementor\Group_Control_Typography;
+use Elementor\Icons_Manager;
+use Elementor\Widget_Base;
+
+trait Button_Trait {
+	/**
+	 * Get button sizes.
+	 *
+	 * Retrieve an array of button sizes for the button widget.
+	 *
+	 * @since 3.4.0
+	 * @access public
+	 * @static
+	 *
+	 * @return array An array containing button sizes.
+	 */
+	public static function get_button_sizes() {
+		return [
+			'xs' => __( 'Extra Small', 'elementor-pro' ),
+			'sm' => __( 'Small', 'elementor-pro' ),
+			'md' => __( 'Medium', 'elementor-pro' ),
+			'lg' => __( 'Large', 'elementor-pro' ),
+			'xl' => __( 'Extra Large', 'elementor-pro' ),
+		];
+	}
+
+	protected function register_button_content_controls( $args = [] ) {
+		$default_args = [
+			'section_condition' => [],
```

Comment:
Add docs for each option (PHPDoc for the $args)

Reply:
Done